{{cookiecutter.project_dirname}}/{{cookiecutter.django_settings_dirname}}/views.py

```diff
@@ -0,0 +1,14 @@
+"""The main app views."""
+
+from django.http import JsonResponse
+from django.views.generic import View
+
+
+class HealthView(View):
+    """The health endpoint view."""
+
+    http_method_names = ("get", "head", "options")
+
+    def get(self, request, *args, **kwargs):
+        """Return health endpoint GET response."""
+        return JsonResponse({"status": "ok"})
```

Comment:
it would be better to return an HttpResponse with no content and status code 204